NitroxClient/Communication/Packets/Processors/InitialPlayerSyncProcessor.cs

```diff
@@ -39,7 +39,7 @@ public override void Process(InitialPlayerSync packet)
         private IEnumerator ProcessInitialSyncPacket(object sender, EventArgs eventArgs)
         {
             // Some packets should not fire during game session join but only afterwards so that initialized/spawned game objects don't trigger packet sending again. 
-            using (PacketSuppressor<PingRenamed>.Suppress())
+            using (PacketSuppressor<EntityMetadataUpdate>.Suppress())
```

Comment:
Did you verify no EntityMetadataUpdate can occure inside this InitialSync which is needed?

Reply:
You're absolutely right asking for this.
While I wouldn't verify if those would happen, I'd rather consider that those must be suppressed in dedicated patches if required. Else, there's no reason to prevent EntityMetadataUpdate from being broadcasted during initial sync.